frontend/src/video-block.js

```diff
@@ -0,0 +1,230 @@
+        >
+          <span class="material-icons">home</span>
+        </button>
+      </Link>
+      <div className="element-wrapper">
+        <div className="iframe-wrapper">
+          <iframe
+            height="200"
+            src={`https://www.youtube.com/embed/${videoId}?rel=0&modestbranding=1&autohide=0&showinfo=0&watch-later=0&controls=captions`}
+            title="YouTube video player"
+            frameborder="0"
+            allowfullscreen="allowFullscreen"
+            className="iframe"
+          ></iframe>
+          <br />
+          <div class="xl video-title">{videoobj.title}</div>
+          <div className="iframe-button">
+            {Like_button(videoobj)}
+
+            <button
+              class="icon-button md"
+              onClick={() => setshowmodal((prev) => !prev)}
+            >
+              <span class="material-icons">playlist_add</span>Save
+            </button>
+          </div>
+        </div>
+        {showmodal === true ? (
+          <PlaylistModal setshowmodal={setshowmodal} videoId={videoId} />
+        ) : (
+          ""
+        )}
+        <div className="recommends">
+          Recommended
+          {data
+            .filter(
+              (item) => item.genre === videoobj.genre && item.id !== videoobj.id
+            )
+            .map((item) => {
+              return (
+                <>
+                  <Link to={`/${item.id}`} className="recommend">
+                    <div className="recommend-flexwrapper">
+                      <div className="image-wrapper">
+                        <img
+                          src={`https://i.ytimg.com/vi/${item.id}/mqdefault.jpg`}
+                          onClick={() => {
+                            dispatch({
+                              type: "ADD_TO_HISTORY",
+                              payload: {
+                                ...item,
+                                ishistory: true,
+                                lastseen: new Date()
+                              }
+                            });
+                          }}
+                          alt="f"
+                        />
+                        <div className="recommendcard-duration md">
+                          {item.duration}
+                        </div>
+                      </div>
+                      <div className="recommendcard-desc">
+                        <div className="recommendcard-title">{item.title}</div>
+                      </div>
+                    </div>
+                  </Link>
+                </>
+              );
+            })}
+        </div>
+      </div>
+    </>
+  );
+}
+
+function PlaylistModal({ setshowmodal, videoId }) {
+  const [newplaylist, setnewplaylist] = useState("");
+  const [showinput, setshowinput] = useState(false);
+
+  let { dispatch, playlist } = useReduce();
+  return (
+    <>
+      <div className="playlistmodal">
```

Comment:
![image](https://user-images.githubusercontent.com/41542536/117571147-d0644480-b0ea-11eb-9b32-c5f4fc16e00b.png)
In the mobile view playlistmodal and the recommended videos are getting overlapped.